src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -135,9 +144,7 @@ public ReadsSparkSource(final JavaSparkContext ctx, final ValidationStringency v
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
     public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, int splitSize) {
```

Comment:
Javadoc for this method still says " This excludes unmapped reads." -- is this still true?